### DISCUSSION TITLE
Trying to understand why there is a 1.1.14 branch and 2.1.5 version of the suite of packages.

### DISCUSSION THREAD

**Discussion by jamiekeefer** [ericsink/SQLitePCL.raw#544] [General]
I was updating some sqlite nuget packages on a couple projects and this issue recently came up. I noticed whoever implemented these has combined 1.1.14 and 2.1.5 package references. I used these to steer to a resolution: 

https://github.com/ericsink/SQLitePCL.raw/issues/476
https://github.com/ericsink/SQLitePCL.raw/issues/384

As I work through these projects, part of a threesome of Xamarin apps, I am updating and resolving build issues - and in short trying to understand why there are two versions of each of these packages. 

looking at the comment: https://github.com/ericsink/SQLitePCL.raw/issues/384
"SQLitePCLRaw 2.x is compatible with sqlite-net-pcl 1.7.
SQLitePCLRaw 1.x is compatible with sqlite-net-pcl 1.6.

So I downgraded all, and with this combination, it works fine (highlighted the changes)"

now that we are at 1.8 on the core sqlite-net-pcl I guess we should always be pairing with 2.X.X branch??

Edit: Is there any reason not to use 2.X.X branch? @ericsink

**Comment by ericsink** [comment]
I *think* your use of the word "branch" here does not correspond to a "Git branch".

My brain is throwing a parse error on "version of the quite of packages" in the title.

Basically, yes, there was a 1.x major version, and then there was a 2.x major version.  They are not compatible.  Nuget packages from the 1.x major version should not be mixed with the 2.x packages.

The last version of the 1.x series was 1.1.14.

You seem to be asking why it still exists.  I'm not sure I understand that.

Oh, wait, some of the packages got new package IDs with the 2.x release.  Maybe that's the confusion.  You're seeing some packages where the latest version is 1.1.14.  That's because that package ID isn't being used by the 2.x series.

I can't think of a reason no to use 2.x series.  Not a likely reason anyway.  I mean, I suppose if you need support for Windows Phone 8 Silverlight , maybe...  But the 1.x series is not maintained.

**Reply by jamiekeefer** [reply]
First, thank you for the prompt reply. My intended use of the word "branch" may have made more sense if I said "fork" but I wasn’t sure what the relationship of the two version series was. I may have used the term "branch" too loosely. Sorry about that. 

Anyway I think you answered the question. In a nutshell I should use the 2.x.x collection of packages to remain up to date today and going forward. These are native Xamarin apps. Android/iOS. Thank you again for the prompt reply.
